pages/common/msfvenom.md

```diff
@@ -0,0 +1,24 @@
+# msfvenom
+
+> Manually generate payloads for metasploit.
+> More information: <https://github.com/rapid7/metasploit-framework/wiki/How-to-use-msfvenom>.
+
+- List Payloads:
+
+`msfvenom -l payloads`
+
+- List formats:
+
+`msfvenom -l formats`
+
+- Show payload options:
+
+`msfvenom -p {{payload}} --list-options`
+
+- Create an elf binary with a reverse tcp handler:
```

Comment:
Shouldn't `elf` be in capital letters, like this?
```suggestion
- Create an ELF binary with a reverse TCP handler:
```

If I do `file /bin/bash`, ELF is in caps. Also, `tcp` should probably be capitalised too.